substratevm/src/com.oracle.svm.core.posix/src/com/oracle/svm/core/posix/JavaUtilZipSubstitutions.java

```diff
@@ -461,6 +652,12 @@ static int update(Object obj, int len, z_stream strm) {
 
 }
 
-/** Dummy class to have a class with the file's name. */
+/**
+ * Dummy class to have a class with the file's name.  This class contains substitutions for {@code java.util.zip}'s
+ * {@code Inflater} and {@code Deflater} classes.  Because both Java 8 and 11 are supported, and these implementations
+ * have differing API, the substitutions contain a mix of things which are shared and which are unique to a specific
+ * version.  Where possible, the origin of the ported code is noted before each substituted method.  The OpenJDK 11
+ * version of these methods are preferred whenever possible.
```

Comment:
All but the first line of this comment belongs on the declaration of `Target_java_util_zip_Inflater`.